packages/framework/src/client.ts

```diff
@@ -49,7 +49,7 @@ JSONSchemaFaker.option({
 });
 
 function isRuntimeInDevelopment() {
-  return process.env.NODE_ENV === 'development';
+  return process.env.NODE_ENV === 'development' || !process.env.NODE_ENV;
```

Comment:
```suggestion
  return ['development', undefined].includes(process.env.NODE_ENV);
```

Reply:
Let's also add a test for this to ensure we don't regress. We have one for `NODE_ENV=development` that we can copy and modify

Reply:
^ this one too @scopsy